app/views/pages/webhooks.html.md

```diff
@@ -0,0 +1,109 @@
+
+* Supplier name
+* Callback URL: this should be an `https://` endpoint which is publicly accessible and should not require authentication
+* Secret: this is a shared secret which is used to generate a SHA-256 HMAC signature to guarantee the authenticity of the notification
+
+Technical note for the Book a Secure Move team to manually create the subscription record:
+
+* connect to the Staging kubernetes pod and in a rails console, run:
+
+        supplier = Supplier.find_by(key: "<SUPPLIER-KEY>")
+        supplier.subscriptions << Subscription.new(callback_url: "<CALLBACK-URL>", secret: "<SECRET>")
+
+Once the Book a Secure Move team have actioned the request, notifications of moves events will be immediately sent to the specified `callback_url`.
+
+
+## 3. Receiving notifications
+The supplier should ensure that the designated endpoint is available to receive notifications at all times. If for any reason the endpoint is offline when a notification is attempted, the PECS system will retry the notification later with an increasing random exponential delay. The system will continue to retry the notification up to 25 times. On the first day the notification will be attempted approximately 14 times (see [this table](https://github.com/mperham/sidekiq/wiki/Error-Handling#automatic-job-retry) for the approximate schedule).
+
+When a notification is received at the supplier's endpoint, it must:
+
+1. Verify the PECS-SIGNATURE header of the notification
+2. Upon verification return an HTTP success code (in the range 200-299)
+
+The notification is not considered to have been delivered until an HTTP success is received.
+
+**_The endpoint should return success (or failure) immediately after signature verification and success should not be contingent on subsequent processing._** 
+
+I.e. if a notification was successfully received and the signature verified, it is not correct to return a failure code because of some other problem in the subsequent processing as this would compromise the decoupled nature of the systems.
+
+An example notification message is given below:
+
+    POST / HTTP/1.1
+    Host: foobar.requestcatcher.com
+    Accept: */*
+    Accept-Encoding: gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+    Connection: keep-alive
+    Content-Length: 415
+    Content-Type: application/vnd.api+json
+    Keep-Alive: 30
+    Pecs-Notification-Id: 2cb108dd-8d47-4a5f-8d36-29324a770f05
+    Pecs-Signature: 5cWQEe9emC7Myvj8jxVDIWI0jxoshOhitXfsCQBtTS4=
+    User-Agent: pecs-webhooks/v1
+    {"data":{"id":"2cb108dd-8d47-4a5f-8d36-29324a770f05","type":"notifications","attributes":{"event_type":"create_move","timestamp":"2020-02-18T11:05:00+00:00"},"relationships":{"move":{"data":{"id":"149f1c27-1b7d-4c60-a4d4-ae8afbe92501","type":"moves"},"links":{"self":"http://hmpps-book-secure-move-api-staging.apps.live-1.cloud-platform.service.justice.gov.uk/api/v1/moves/149f1c27-1b7d-4c60-a4d4-ae8afbe92501"}}}}}
```

Comment:
Could this JSON blob be pretty-printed to ease comprehension? Not at all crucial, as the response is compact, and the pretty structure is shown below.